textattack/attack_metrics/attack_metric.py

```diff
@@ -0,0 +1,27 @@
+"""
+Attack Metrics Class
+========================
+
+"""
+
+from abc import ABC, abstractmethod
+
+from textattack.attack_results import AttackResult
+
+
+class AttackMetric(AttackResult, ABC):
```

Comment:
What's the reason why `AttackMetric` is a sub-class of `AttackResult`?

Reply:
@sanchit97 I have the same concerns as @jinyongyoo above

Reply:
Yes, this is an oversight. Fixed.